teuthology/orchestra/opsys.py

```diff
@@ -159,6 +159,35 @@ def from_os_release(cls, os_release_str):
 
         return obj
 
+
+    @classmethod
+    def version_codename(cls, name, version_or_codename):
+        '''
+        Set version and codename based on one input, trying to infer
```

Comment:
The code doesn't "set" anything; just returns values.

Also can we use ``"""`` instead of ``'''`` please?